src/components/pages/home/Home.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import "./home.css";
+import { categories } from "data/category-data";
+import { 
+    Card, 
+    CardCategory 
+} from "components";
+export const Home = () => {
+  return (
+    <main>
+      <section className="landing-page">
+        <div className="content">
+          <h1>Learning that gets you</h1>
+          <p>Skills for your present (and your future). Get started with us.</p>
+        </div>
+        <div className="represent-img">
+          <img
+            loading="lazy"
```

Comment:
if this image is on your landing page then adding `loading="lazy" ` is not good.